add_to_app/multiple_flutters_ios/MultipleFluttersIos/AppDelegate.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+import Flutter
+
+/// The app's UIApplicationDelegate.
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+  let engines = FlutterEngineGroup(name: "multiple-flutters", project: nil)
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    #if DEBUG
+        let isDebug = true
+    #else
+        let isDebug = false
+    #endif
+    if (isDebug) {
+      NSLog("⚠️ WARNING: the memory and CPU costs for Flutter engine groups are significantly greater in debug builds.  See also: https://github.com/dart-lang/sdk/issues/36097")
+    } else {
+      NSLog("⚠️ WARNING: the memory and CPU costs for Flutter engine groups are significantly less here than in debug builds.  See also: https://github.com/dart-lang/sdk/issues/36097")
```

Comment:
What do you like better?  "NOTICE"?

Reply:
I switched it to NOTICE with a megaphone emoji

Reply:
Sorry, I just meant that for debug mode, we should show a warning or something scary because the whole thing is about performance and performance is crappy on debug. 

For non-debug where it's working, it doesn't have to be a scary prompt or any messaging that sounds like they need to do anything about it.